localnet/scripts/run.sh

```diff
@@ -136,16 +137,16 @@ function wait_for_localnet_boot() {
     result=$(curl --silent --location --request POST "localhost:9500" \
       --header "Content-Type: application/json" \
       --data '{"jsonrpc":"2.0","method":"hmy_blockNumber","params":[],"id":1}' | jq '.result')
-    if [ "$result" = "\"0x0\"" ]; then
+    if ((result>0)); then
```

Comment:
I just tested your code and i don't think this condition will ever work. 
```
Waiting for localnet to boot...
Waiting for localnet to boot...
Waiting for localnet to boot...
Waiting for localnet to boot...
^C
```
why not just doing
```bash
if [ "$result" != "\"0x0\"" ]; then 
```
this should work better, basically testing if the block number is not 0.

Reply:
I think I missed the -r flag